linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/DashboardHandler.scala

```diff
@@ -45,3 +48,7 @@ private[admin] class DashboardHandler extends Service[Request, Response] {
     )
   }
 }
+
+object DashboardHandler {
+  val rexp = "/routers/(.*)/?".r
```

Comment:
do we need ^ and $ anchors so that this doesn't match longer urls that include this?